solverdummy/README.md

```diff
@@ -1,8 +1,6 @@
 # MATLAB solverdummy
 
-This dummy illustrates the use of precice with MATLAB using the MATLAB bindings based on the C data API.
-
-**Please note:** Currently, the MATLAB solverdummy can't be coupled with other solvers unless an exchange directory is specified with its absolute path in the config file. So, if you wish to couple with a different solver, please make sure to modify the config file accordingly.
```

Comment:
You fixed this? Nice! Then maybe also add a changelog entry for this.

Reply:
I couldn't recreate the error described, so as you suggested in #18, this was probably a documentation error.